react-app/src/components/ProfileIcon/ProfileIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Volunteer } from "../../types/UserType.ts";
+
+interface ProfileIconProps {
+  user: Volunteer;
+}
+
+const ProfileIcon: React.FC<ProfileIconProps> = ({ user }) => {
+  const initials = `${user.firstName.charAt(0)}${user.lastName.charAt(0)}`;
+
+  return (
+    <div
```

Comment:
Move all the styling for this page to its own CSS file.

Reply:
Fixed.